plugins/kotlin-dataframe/testData/box/groupBy_count.kt

```diff
@@ -0,0 +1,13 @@
+import org.jetbrains.kotlinx.dataframe.*
+import org.jetbrains.kotlinx.dataframe.annotations.*
+import org.jetbrains.kotlinx.dataframe.api.*
+import org.jetbrains.kotlinx.dataframe.io.*
+
+fun box(): String {
+    val df = dataFrameOf("a")(1, 1, 2, 3, 3).groupBy { a }.count()
```

Comment:
can you add a test for count("otherName") as well? :)

Reply:
sure